docs/guides/admin/docs/upgrade.md

```diff
@@ -48,6 +48,21 @@ Please make sure to compare your configuration against the current configuration
   [this document](https://github.com/elan-ev/opencast-studio/blob/2020-09-14/CONFIGURATION.md).
 
 
+### Wowza streaming configuration changes
+
+In [this pull request](https://github.com/opencast/opencast/pull/1179), the configuration of Wowza servers was made
+tenant-specific. Before, you had to specify `org.opencastproject.streaming.url` in `custom.properties`. Starting with
+Opencast 9, that value has to specified as `org.opencastproject.mh_default_org.wowza.url` in
+`org.opencastproject.distribution.streaming.wowza.WowzaStreamingDistributionService.cfg`. Of course, if you have
+multiple tenants, you want to replace `mh_default_org` in that key accordingly.
```

Comment:
```suggestion
Opencast 9, it is now a tenant-specific configuration in `etc/org.opencastproject.distribution.streaming.wowza.WowzaStreamingDistributionService.cfg`.
```

Reply:
I removed the "Of course" sentence in the end, but kept my version of the first sentence as I think the key name is worth mentioning and my version avoid the repetition of "tenant specific".